### ISSUE TITLE
[stable/airflow] Airflow on Minikube

### ISSUE THREAD

**Issue by JWDobken** [helm/charts#16257]
**Describe the bug**
I have issues with installing Airflow on minikube, after installing the helm chart with:

    helm install --namespace "airflow" --name "airflow" stable/airflow

When everything is initialized, `kubectl get po -n airflow` returns:

```sh
NAME                                 READY   STATUS    RESTARTS   AGE
airflow-flower-7b4cbb6ff5-j9wph      0/1     Error     1          45s
airflow-postgresql-bdcb64f8d-f4qgb   1/1     Running   0          45s
airflow-redis-master-0               1/1     Running   0          45s
airflow-scheduler-86d6d9f5d-6fdg5    0/1     Error     1          44s
airflow-web-67cf9cf59c-2x8t6         1/1     Running   0          45s
airflow-worker-0                     1/1     Running   0          45s
```

The logs of the broken scheduler and the flower give the same issue with werkzeug:

```sh
...
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 69, in <module>
    from .wrappers import Request
  File "/usr/local/lib/python3.6/site-packages/flask/wrappers.py", line 14, in <module>
    from werkzeug.wrappers.json import JSONMixin as _JSONMixin
ModuleNotFoundError: No module named 'werkzeug.wrappers.json'; 'werkzeug.wrappers' is not a package
```
This led me to the following issue <https://github.com/puckel/docker-airflow/issues/395>, which is solved by rolling back the version of `puckel/docker-airflow` to version `1.10.2`

But now the web pod is broken:

```sh
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 87, in create
    dbapi = dialect_cls.dbapi(**dbapi_args)
  File "/usr/local/lib/python3.6/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 599, in dbapi
    import psycopg2
  File "/usr/local/lib/python3.6/site-packages/psycopg2/__init__.py", line 50, in <module>
    from psycopg2._psycopg import (                     # noqa
ImportError: libpq.so.5: cannot open shared object file: No such file or directory
```

This leads me to <https://github.com/puckel/docker-airflow/issues/342> where is the advice to update tversion of `puckel/docker-airflow` to version `1.10.3`.

minikube version: v1.3.0
Helm: v2.14.3
kubernetes client version: v1.15.2
kubernetes server version: v.15.2

**Comment by JWDobken** [comment]
hee rolling forward to version `puckel/docker-airflow:1.10.4` solves the issue

**Comment by saden1** [comment]
+1

I am seeing the issue in a k8 cluster.

**Comment by saden1** [comment]
Issue has been fixed in Airflow v1.10.4, see https://issues.apache.org/jira/browse/AIRFLOW-4900

**Comment by orkenstein** [comment]
@JWDobken were you able to run this chart correctly?
